math/montecarlo/estimation_e.go

```diff
@@ -0,0 +1,42 @@
+package montecarlo
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+	"time"
+)
+
+const (
+	e = math.E
+)
+
+func estimation_e() {
```

Comment:
This is an untestable implementation. Please write code that you can easily test. Refer to [How to contribute](https://github.com/TheAlgorithms/Go/blob/master/CONTRIBUTING.md) and look at some other code within this repo to see how to do it.

Reply:
Sure, thank you.